src/redux/books/apiThunks.js

```diff
@@ -0,0 +1,31 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import {
+  addBookToAPI,
+  getStateFromAPI,
+  removeBookFromAPI,
+} from '../../component/api/actioApi';
+import {
+  GET, ASYNC_ADD, ASYNC_REMOVE, addBook, removeBook,
+} from './apiBooks';
+
+const asyncAdd = createAsyncThunk(ASYNC_ADD, async (book, { dispatch }) => {
+  const answer = await addBookToAPI(book);
+  dispatch(addBook(book));
+  return answer;
+});
+
+const asyncRemove = createAsyncThunk(
+  ASYNC_REMOVE,
+  async (book, { dispatch }) => {
+    const answer = await removeBookFromAPI(book);
+    dispatch(removeBook(book));
+    return answer;
+  },
+);
+
+const asyncGet = createAsyncThunk(GET, async () => {
+  const answer = await getStateFromAPI();
+  return answer.books;
+});
+
```

Comment:
![Capture2](https://user-images.githubusercontent.com/93039359/200511917-b29322a8-c52e-494c-ab0e-3afdefb5e329.PNG)

- [ ] You have missed these requirements 😨 Kindly add this file outside of the books folder 😊 The correct path for this file is `src/redux/apiThunks.js` 👍